src/netmanager-app/components/Analytics/index.tsx

```diff
@@ -23,19 +23,19 @@ const NewAnalytics: React.FC = () => {
   const [downloadingData, setDownloadingData] = useState(false)
   const [activeGrid, setActiveGrid] = useState<Grid>()
   const [activeCohort, setActiveCohort] = useState<Cohort>()
-  const [transformedReadings, setTransformedReadings] = useState<{
-    type: string
-    features: {
-      type: "Feature"
-      properties: unknown
-      geometry: { type: "Point"; coordinates: [number, number] }
-    }[] // Updated type
-  } | null>(null)
+  // const [transformedReadings, setTransformedReadings] = useState<{
+  //   type: string
```

Comment:
Hi @Codebmk , do we need this commented code for the future?

Reply:
Yeap it is still needed